doc/vim-plugin.md

```diff
@@ -328,3 +331,63 @@ Use `<tab>` to toggle selection and CTRL-A/CTRL-D to select all/select none.
 See the
 [Fzf](https://github.com/junegunn/fzf) documentation for more details.
 
+Quickfix List
+-------------
+
+The quickfix list is used to show a list of positions in files and access them
+quickly. Phpactor use it for example to show the result of `find references`.
+
+### Strategies
+
+This plugin provides two strategy to handle input lists:
+
+- `vim`: Vim's basic quickfix.
+- `fzf`: Use [fzf](#fzf) to show allow you to filter the results.
+
+With the [fzf](#fzf) strategy you will still be able to get the result inside
+the quickfix by selecting the elements you are interested in and pressing
+`ctrl-q` to populate the quickfix with your selection and open it.
+
+You can use your own strategy by providing a `Funcref` to the
+`g:phpactorQuickfixStrategy` variable. When calling a strategy the list of
+positions is given as a list that is directly usable by the function
+`setqflist()`. Not all informations are provided, depending on the context, so
```

Comment:
`informations are` => `information is`. Not sure what is meant by `information` here (assume it relates to the hashmap that's passed to the func?). Maybe better to ommit this and just have it as a hidden feature for now? Otherwise it would be better to provide an example.

Reply:
Maked it `not all the keys are provided`
Because I'm afraid that just `information` is to generic and won't reflect properly what might be missing.